core/src/main/java/nl/nn/adapterframework/scheduler/JobDefFunctions.java

```diff
@@ -60,7 +60,7 @@ public boolean isNotEqualToAtLeastOneOf(JobDefFunctions... functions) {
 	public boolean isEqualToAtLeastOneOf(JobDefFunctions... functions) {
 		boolean equals = false;
 		for (JobDefFunctions func : functions) {
-			if(super.equals(func))
+			if(super == func)
```

Comment:
Using `super` on its own is not legal indeed. `super` should be replaced with `this`.

And the function can also be optimised to `return true` instead of assigning `equals = true`

Reply:
Yes, thanks @tnleeuw. Just fixed it and improved the code around it.